packages/aquedux-server/src/redis/asyncSnapshotQueue.js

```diff
@@ -7,39 +7,46 @@ import actions from '../actions'
 import { selectors } from '../reducers'
 import logger from '../utils/logger'
 
-// TODO: Check if this must be used.
-//
-// const luaScript = `
-// local prefix = ARGV[1]
-// local limit = tonumber(ARGV[2])
-// local action = ARGV[3]
+const { queueLimit } = configManager.getConfig()
+/*
+const luaScript = `
+local prefix = ARGV[1]
+local limit = tonumber(ARGV[2])
+local action = ARGV[3]
 
 // local latestFragmentIndex = redis.call('get', prefix .. '-head')
 // local latestFragmentSnapshotName = prefix .. '-snap-' .. latestFragmentIndex
 // local latestFragmentSnapshot = redis.call('get', latestFragmentSnapshotName)
 
-// if latestFragmentSnapshot == nil then
-//     local nextFragment =  prefix .. '-frag-' .. (latestFragmentIndex + 1)
-//     redis.call('rpush', nextFragment, action)
-//     redis.call('incr', prefix .. '-head')
-//     return 1
-// else
-//     redis.call('rpush', latestFragment, action)
-//     return 0
-// end
-// `
+if limit > 0 and latestFragmentSnapshot == nil then
+    local nextFragment =  prefix .. '-frag-' .. (latestFragmentIndex + 1)
+    redis.call('rpush', nextFragment, action)
+    redis.call('incr', prefix .. '-head')
+    return 1
+else
+    redis.call('rpush', latestFragment, action)
+    return 0
+end
+`*/
 
 const asyncSnapshotQueue = async (store: Store, name: string, size: number) =>
   asyncQuery(async connection => {
+    const { doFragmentSnapshot } = configManager.getConfig()
+    if (!doFragmentSnapshot) {
+      logger.warn({
+        who: `redis-${name}`,
+        what: 'doFragmentSnapshot method not defined. Skipping asyncSnapshotQueue...'
+      })
+      return
+    }
```

Comment:
Should we? In the configuration, we have a default implementation, and it should work. Or else it is a mandatory configuration key and should be set to null by default (if not this `if(!doFragmentSnapshot)` would not trigger at all).